deployment/helm/node-feature-discovery/templates/topologyupdater.yaml

```
@@ -66,8 +66,8 @@ spec:
           - "-key-file=/etc/kubernetes/node-feature-discovery/certs/tls.key"
           - "-cert-file=/etc/kubernetes/node-feature-discovery/certs/tls.crt"
           {{- end }}
-          {{- if .Values.topologyUpdater.podSetFingerprint }}
-          - "-pods-fingerprint"
+          {{- if .Values.topologyUpdater.podSetFingerprint | empty }}
```

nit: I'm not sure if there's a functional difference but but using `not` would feel more correct for boolean args

```suggestion
          {{- if not .Values.topologyUpdater.podSetFingerprint }}
```

---

yes, there's no functional difference, not condition is more clear
done